package.json

```diff
@@ -12,11 +12,11 @@
         "test": "npx qunit-testrunner",
         "test:keepAlive": "npx qunit-testrunner --keepalive",
         "test:cov": "npm run build:cov && npx qunit-testrunner --cov",
-        "test:dev": "NODE_ENV=development run-p test:keepAlive build:watch",
+        "test:dev": "cross-env NODE_ENV=development run-p test:keepAlive build:watch",
```

Comment:
Is this addition related to the feature? If no, you should rather add it in a separate PR